src/User.js

```diff
@@ -1,16 +1,16 @@
-import UserRepository from './UserRepository';
-import userData from './data/users';
-let userRepository = new UserRepository(userData) // Get rid of this eventually
+// import UserRepository from './UserRepository';
+// import userData from './data/users';
+// let userRepository = new UserRepository(userData); // Get rid of this eventually
 
 class User {
-  constructor(user) {
-    this.id = this.checkUserId(user.id);
-    this.name = this.checkName(user.name);
-    this.address = user.address || 'No address added.';
-    this.email = user.email || 'No email address added.';
-    this.strideLength = user.strideLength || 'Stride length not added.';
-    this.dailyStepGoal = user.dailyStepGoal || 'Daily step goal not added.';
-    this.friends = user.friends || 'Add friends for friendly competition!';
+  constructor(userDetails) {
+    this.id = this.checkUserId(userDetails.id);
+    this.name = this.checkName(userDetails.name);
+    this.address = userDetails.address || 'No address added.';
+    this.email = userDetails.email || 'No email address added.';
+    this.strideLength = userDetails.strideLength || 'Stride length not added.';
+    this.dailyStepGoal = userDetails.dailyStepGoal || 'Daily step goal not added.';
+    this.friends = userDetails.friends || 'Add friends for friendly competition!';
     this.ouncesAverage = 0;
     this.ouncesRecord = [];
```

Comment:
We talked a bit about renaming this so let's hit that on the next standup. Even recordOfOunces would be acceptable

Reply:
Yeah I agree we should rename this.